### ISSUE TITLE
If the following row has a midrule, remove the previous addlinespace

### ISSUE THREAD

**Issue by jamesfeigenbaum** [jamesfeigenbaum/textablr#3]
If the following row has a midrule, remove the previous addlinespace

This is low priority and is only about latex formatting

**Comment by jamesfeigenbaum** [comment]
A little kludgy, but this works now. Removed from the tex code but only after sending the tex code to become a markdown table...
